src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -955,7 +955,7 @@ export type BaseGroup = {
   latestEvent: Event;
   activity: GroupActivity[];
   annotations: string[];
-  assignedTo: User;
+  assignedTo: Actor;
```

Comment:
assignedTo is an Actor, was only passing before because createReactClass cast this.state and this.props to any type.